tools/code-analyzer/src/commands/analyzer/index.ts

```diff
@@ -233,15 +233,11 @@ export default class Analyzer extends Command {
 	) {
 		const { output, file } = flags;
 		CliUx.ux.action.start( 'Generating changes' );
+
 		const templates = this.scanTemplates( content, version );
 		const hooks = this.scanHooks( content, version, output );
 		const databaseUpdates = this.scanDatabases( content );
-
-		await generateJSONFile( join( process.cwd(), file ), {
-			templates: Object.fromEntries( templates.entries() ),
-			hooks: Object.fromEntries( hooks.entries() ),
-			schema: databaseUpdates || {},
-		} );
+		let schemaDiffResult = {};
```

Comment:
I wouldn't normally declare a let variable like this for collecting the schema change, but I had to do some refactoring of `printSchemaChange` to get the data I needed. I think `code-analyzer` needs a bit more work to be more generally useful and I've logged an issue for this on the Atlas team board.